### DISCUSSION TITLE
Add Categories/Labels to Addy.io Aliases

### DISCUSSION THREAD

**Discussion by jaainil** [anonaddy/anonaddy#657] [Feature Requests]
**Description:**
Implement a system to categorize or label email aliases in Addy.io, allowing users to easily identify which alias is associated with specific services or purposes.

**Key Points:**

1. Add a "Category" or "Label" field for each alias
2. Allow users to create, edit, and delete custom categories
3. Enable filtering and sorting aliases by category
4. Include category information in the alias list view

**Benefits:**

- Improved organization of aliases
- Easier management of multiple aliases
- Quicker identification of alias purposes
- Enhanced user experience for those with many aliases (like me 🥲)

**Comment by willbrowningme** [comment]
Thanks, this is currently on the roadmap - https://github.com/anonaddy/anonaddy/issues/574
